public/style.css

```diff
@@ -1,34 +1,41 @@
+@import url(http://fonts.googleapis.com/css?family=Source+Sans+Pro:200,300);
```

Comment:
It gives the following warning ` The page at 'https://chat.sdslabs.co/' was loaded over HTTPS, but requested an insecure stylesheet 'http://fonts.googleapis.com/css?family=Source+Sans+Pro:200,300'. This content should also be served over HTTPS.`